internal/format/json.go

```diff
@@ -29,7 +29,7 @@ func NewJSON(settings *print.Settings) print.Engine {
 
 // Print a Terraform module as json.
 func (j *JSON) Print(module *terraform.Module, settings *print.Settings) (string, error) {
-	copy := &terraform.Module{
+	sections := &terraform.Module{
```

Comment:
Revert this back to `copy` to give clear indication that this is a _copy_ of `module`.